frontend/src/DeviceDetailsPage/TabPanels/DeviceDetailsActivityPanel.jsx

```diff
@@ -0,0 +1,6 @@
+import { Container } from "@material-ui/core";
+import React from "react";
+
+export default function DeviceDetailsActivityPanel() {
+  return <Container>TODO: Design activity log</Container>;
+}
```

Comment:
It's not a defined page. There is no mockup. It's here because the tab is indicated but the contents is undefined. 
It's here because otherwise, I don't have a way to demo that the tabs work. And we will want this file to exist later.